File: app/src/main/java/com/mirogal/cocktail/data/repository/NetworkState.java
package com.mirogal.cocktail.data.repository;

public class NetworkState {

    public static final Status EMPTY = Status.SUCCESS_EMPTY;
    public static final Status LOADED = Status.SUCCESS_LOADED;
    public static final Status LOADING = Status.RUNNING;

    public enum Status {
        RUNNING,
        SUCCESS_LOADED,
        SUCCESS_EMPTY,
        //TODO (FAILED status processing)
    }
}

File: app/src/main/java/com/mirogal/cocktail/data/repository/NetworkState.kt
package com.mirogal.cocktail.data.repository

object NetworkState {

    val EMPTY = Status.SUCCESS_EMPTY
    val LOADED = Status.SUCCESS_LOADED
    val LOADING = Status.RUNNING

    enum class Status {
        RUNNING, SUCCESS_LOADED, SUCCESS_EMPTY
        //TODO (FAILED status processing)
    }

}